packages/iTwinUI-react/src/core/Wizard/WorkflowDiagram.tsx

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils';
+import '@itwin/itwinui-css/css/workflow-diagram.css';
+import { StepProperties } from './Wizard';
+import { Step } from './Step';
+
+export type WorkflowDiagramProps = {
+  /**
+   * An array of step objects.
+   */
+  steps: StepProperties[];
+  /**
+   *  Click handler on completed step.
+   */
+  onStepClick?: (clickedIndex: number) => void;
```

Comment:
Aren't workflow steps not clickable at all?

Reply:
Removed `onStepClick` from `WorkflowDiagram`. I thought there were cases where users could pass `type: 'workflow'` and pass an `onStepClick` function into `Wizard` causing a breaking change

Reply:
So users will still be able to pass onClick handler to the Wizard but it won't do anything as before.